### DISCUSSION TITLE
Synconize two players using Slimpro (LMS)

### DISCUSSION THREAD

**Discussion by rysm83** [music-assistant/support#3142] [Q&A]
I have two Android tablets and initially tried to sync them using Snapcast. After installing the Snapcast app on both, I was able to get music playing to them through Music Assistant (MA), and they did sync. Now, I want to try the Slimproto or Squeeze option, as the MA webpage suggests it offers more options and near-perfect sync.

I installed the Squeeze player from the app store and was able to connect to the MA server. I can play music to each individual player without issues, but I can't get them to sync. The sync option is available, but no music plays on the player I'm trying to sync to. I also noticed a glitch: when the players are synced, and I start a song or playlist, the progress counter starts, then resets to 0, repeating this cycle until I manually move it forward by about 15 seconds. After that, the music plays, but only on one player.

I also tried creating a group using the Slimproto and selected both players, but I’m getting the same result. I wanted to confirm that I'm not doing anything wrong on my end.

**Comment by OzGav** [comment]
Please don’t comment in two places. I see you found the SB player issue. If you are using that then you know the problem.
